src/test/java/com/android/tools/build/bundletool/model/utils/files/FilePreconditionsTest.java

```diff
@@ -22,13 +22,15 @@
 import static com.android.tools.build.bundletool.model.utils.files.FilePreconditions.checkFileExistsAndReadable;
 import static com.android.tools.build.bundletool.model.utils.files.FilePreconditions.checkFileHasExtension;
 import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assume.*;
```

Comment:
Please remove the wildcard and use the full name of the method:
https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports